spec/correios_sigep/configuration_spec.rb

```diff
@@ -80,5 +80,18 @@ module CorreiosSigep
         expect(subject.user).to eq 'user'
       end
     end
+
+    describe '#timeout' do
+      it 'return nil when unset' do
+        expect(subject.timeout).to_not be
```

Comment:
@fpgentil What u think about change all `to_not be` to `not_to be`?
I think it is more english readable 😄